static/scss/includes/datatables.scss

```diff
@@ -92,6 +92,16 @@ table caption {
   table.dataTable {
     padding: 20px 0;
 
+    // reward!
+    .great {
```

Comment:
Can this also be renamed to `.success` or something similar? It actually took me a few seconds to realize in the JS that great in this context doesn't mean bolder or larger.